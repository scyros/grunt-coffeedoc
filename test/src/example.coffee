###
# CoffeeDoc example documentation #

This is a module-level docstring, and will be displayed at the top of the module documentation.
Documentation generated by [CoffeeDoc](http://github.com/omarkhan/coffeedoc)
###

class MyClass extends Superclass
    ###
    This docstring documents MyClass. It can include *Markdown* syntax,
    which will be converted to html.
    ###
    constructor: (@args) ->
        ### Constructor documentation goes here. ###

    method: (args) ->
        ### This is a method of MyClass ###

myFunc = (arg1, arg2, args...) ->
    ###
    This function will be documented by CoffeeDoc
    ###
    doSomething()
