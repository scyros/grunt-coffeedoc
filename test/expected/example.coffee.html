<!doctype html>

<!-- Documentation generated by [CoffeeDoc](http://github.com/omarkhan/coffeedoc) -->

<html>
<head>
  <title>CoffeeDoc | example.coffee</title>
  <meta http-equiv="content-type" content="text/html; charset=UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" media="all" href="../../resources/base.css" />
</head>
<body>
  <header>
    <h1>example.coffee</h1>
  </header>
  <div class="container">
    <div class="sidebar column">

      <h4>Classes</h4>
      <ul class="classlist">

        <li><a href="#MyClass">MyClass</a></li>

      </ul>


      <h4>Functions</h4>
      <ul class="functionlist">

        <li><a href="#myFunc">myFunc</a></li>

      </ul>

    </div>
    <div class="content column">

      <div id="module-docstring">
        <h1 id="coffeedoc-example-documentation">CoffeeDoc example documentation</h1>
<p>This is a module-level docstring, and will be displayed at the top of the module documentation.
Documentation generated by <a href="http://github.com/omarkhan/coffeedoc">CoffeeDoc</a></p>

      </div>


      <div id="classes">
        <h2>Classes</h2>

        <div class="class">
          <div class="header">
            <a id="MyClass"><h3>MyClass</h3></a>


            <a class="parent" href="#Superclass"><h3>extends Superclass</h3></a>


          </div>
          <div class="class-content">
            <p>This docstring documents MyClass. It can include <em>Markdown</em> syntax,
which will be converted to html.</p>

            <div class="methods">


              <div class="instancemethods">
                <h3>Instance Methods</h3>

                <div class="method">
                  <h4>constructor(@args)</h4>
                  <p>Constructor documentation goes here. </p>

                </div>

                <div class="method">
                  <h4>method(args)</h4>
                  <p>This is a method of MyClass </p>

                </div>

              </div>


            </div>
          </div>
        </div>

      </div>


      <div id="functions">
        <h2>Functions</h2>

        <div class="function">
          <div class="header">
            <a id="myFunc"><h3>myFunc(arg1, arg2, args...)</h3></a>
          </div>

          <div>
            <p>This function will be documented by CoffeeDoc</p>

          </div>

        </div>




      </div>

    </div>
  </div>
</body>
</html>
